src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -1362,4 +1366,23 @@ internal static void Copy(this IInteractiveWindowOperations operations)
             ((IInteractiveWindowOperations2)operations).Copy();
         }
     }
+
+    internal sealed class TestClipboard : InteractiveWindowClipboard
+    {
+        DataObject _data = new DataObject();
+
+        internal void Clear() => _data = new DataObject();
+
+        internal IDataObject GetDataObject() => _data;
+
+        internal override bool ContainsData(string format) =>  _data?.GetData(format) != null;
+
+        internal override object GetData(string format) => _data?.GetData(format);
+
+        internal override bool ContainsText() => _data != null ? _data.ContainsText() : false;
+
+        internal override string GetText() => _data?.GetText();
+
+        internal override void SetDataObject(object data, bool copy) => _data = (DataObject)data;
+    }
```

Comment:
@cston Yeah, I just had a closer look at the Clipboard API, SetData/Text/etc methods do expect a non-null argument, so I guess what you suggested would be a more accurate mock. Will fix. Thanks!